react/features/participants-pane/components/breakout-rooms/components/native/CollapsibleList.js

```diff
@@ -0,0 +1,65 @@
+// @flow
```

Comment:
Take this file out of the breakout-rooms subdirectory please, since it's now generic for the panel elements.